nemo/collections/asr/parts/mixins/mixins.py

```diff
@@ -412,6 +412,9 @@ def change_attention_model(
             update_config (bool): Whether to update the config or not with the new attention model.
                 Defaults to True.
         """
+        if self_attention_model is None and att_context_size is None:
+            return
```

Comment:
Don't we want to reset the attention to full context ?

Reply:
no, in this method setting either parameter to None means not changing it (this is default so you can just change one of them), so if both are None we can just return